incubator/hnc/Makefile

```diff
@@ -91,22 +76,50 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
-# Creates a local kind cluster, destroying the old one if necessary. It's not
-# *necessary* to call this wih CONFIG=kind but it's not a bad idea either so
-# the correct manifests get created.
-kind-reset: manifests
-	@echo "Warning: you must have sourced 'devenv' so kubectl points to the Kind cluster"
+###################### DEPLOYABLE ARTIFACTS AND ACTIONS #########################
+
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: deploy-prereq docker-push kubectl manifests
+	-kubectl -n hnc-system delete deployment hnc-controller-manager
```

Comment:
Any reason not to use `kubectl delete -f manifests/hnc-manager.yaml` here?

Reply:
Good call, fixed.

Reply:
Actually, this deletes the CRDs so any existing hierarchy configs are wiped out too. Oops.

Reply:
I'm reverting this back to the way it was.

Reply:
I updated the comment now, lgty?

Reply:
👍